test/dummy/config/initializers/devise.rb

```diff
@@ -1,17 +1,290 @@
 # frozen_string_literal: true
 
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
-  config.authentication_keys = [:email, :nickname]
+  # The secret key used by Devise. Devise uses this key to generate
```

Comment:
do we ned to change this file? I don't know if we need to add this documentation (I know it's in the default devise initializer )

Reply:
I think we need. It is only for test application (play with settings easily while testing).